wp-tests-config-sample.php

```diff
@@ -19,12 +19,12 @@
 // Tests with an associated Trac ticket that is still open are normally skipped.
 // define( 'WP_TESTS_FORCE_KNOWN_BUGS', true );
 
-// Test with WordPress debug mode (default).
+// Test with ClassicPress debug mode (default).
 define( 'WP_DEBUG', true );
 
 // ** MySQL settings ** //
 
-// This configuration file will be used by the copy of WordPress being tested.
+// This configuration file will be used by the copy of ClassicPress being tested.
 // wordpress/wp-config.php will be ignored.
```

Comment:
Shouldn't `wordpress` be changed to `classicpress` in this instance?

Reply:
I think this should be `src/wp-config.php` instead.